stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -315,6 +315,7 @@ BitMask BuiltinTypeInfo::getSpareBits(TypeConverter &TC, bool &hasAddrOnly) cons
     mask.keepOnlyMostSignificantBits(getSize() * 8 - intSize);
     return mask;
   } else if (
+    Name == "ypXp" || // Any.Type
```

Comment:
There are doubtless other pointer types that need to expose spare bits.  I expect some further iteration of this specific test here.